File: python/Face/rest/detect.py
import json, os, requests

subscription_key = "PASTE_YOUR_FACE_SUBSCRIPTION_KEY_HERE"

face_api_url = "PASTE_YOUR_FACE_ENDPOINT_HERE" + '/face/v1.0/detect'

image_url = 'https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/faces.jpg'

headers = {'Ocp-Apim-Subscription-Key': subscription_key}

params = {
    'detectionModel': 'detection_02',
    'returnFaceId': 'true'
}

response = requests.post(face_api_url, params=params,
                         headers=headers, json={"url": image_url})
print(json.dumps(response.json()))

File: javascript/Face/rest/detect.js
// <environment>
'use strict';

const axios = require('axios').default;

let subscriptionKey = 'PASTE_YOUR_FACE_SUBSCRIPTION_KEY_HERE'
let endpoint = 'PASTE_YOUR_FACE_ENDPOINT_HERE' + '/face/v1.0/detect'

// Optionally, replace with your own image URL (for example a .jpg or .png URL).
let imageUrl = 'https://raw.githubusercontent.com/Azure-Samples/cognitive-services-sample-data-files/master/ComputerVision/Images/faces.jpg'
// </environment>

// <main>
// Send a POST request
axios({
    method: 'post',
    url: endpoint,
    params : {
		detectionModel: 'detection_03',
        returnFaceId: true
    },
    data: {
        url: imageUrl,
    },
    headers: { 'Ocp-Apim-Subscription-Key': subscriptionKey }
}).then(function (response) {
    console.log('Status text: ' + response.status)
    console.log('Status text: ' + response.statusText)
    console.log()
    console.log(response.data)
}).catch(function (error) {
    console.log(error)
});
// </main>
